ci/do_ci.sh

```diff
@@ -95,7 +95,7 @@ if [[ $# -gt 1 ]]; then
 else
   # Coverage test will add QUICHE tests by itself.
   COVERAGE_TEST_TARGETS=//test/...
-  TEST_TARGETS="${COVERAGE_TEST_TARGETS} $(bazel query ${BAZEL_QUERY_OPTIONS} "attr('tags', 'test_included', '@com_googlesource_quiche//:all')" | grep "^@com_googlesource_quiche")"
+  TEST_TARGETS="${COVERAGE_TEST_TARGETS} $(bazel query ${BAZEL_QUERY_OPTIONS} "tests(@com_googlesource_quiche//:ci_tests)")"
```

Comment:
I don't think you need bazel query then, you can directly use it at test target?